### ISSUE TITLE
Cannot Add Account

### ISSUE THREAD

**Issue by robeyl** [rmcrackan/Libation#819]
Hi, first time using Libation.

I have installed the latest version (11.3.0) Chardonnay on MacOS and on Ubuntu 23.10.

However, on both systems, when I open the Accounts window to add an account, I am unable to enter anything into the text fields.
I can toggle the Locale drop-down, but that's about it.

![CleanShot 2024-01-04 at 10 45 46@2x](https://github.com/rmcrackan/Libation/assets/7686002/ff566c8e-464b-4775-911a-7e21a9e5cacb)

**Comment by nhurden** [comment]
Also getting this on 11.3, 11.1 works for me though.

**Comment by robeyl** [comment]
> Also getting this on 11.3, 11.1 works for me though.

@nhurden thanks for the tip! Using 11.1 for now, working for me.

**Comment by rmcrackan** [comment]
So -- 11.1 works for mac and linux and 11.3 doesn't? Damn, I thought I had it this time. Time to revert back to 11.1 again :(

**Comment by CLHatch** [comment]
> So -- 11.1 works for mac and linux and 11.3 doesn't? Damn, I thought I had it this time. Time to revert back to 11.1 again :(

I just checked, and the Add Account issue appears to be an issue in Windows Chardonnay also.  Doesn't let me type in an account name.

**Comment by rmcrackan** [comment]
I believe I found this bug. Can you try [v11.3.1](https://github.com/rmcrackan/Libation/releases/tag/v11.3.1) and see if this fixes it for you?

**Comment by robeyl** [comment]
@rmcrackan Confirmed fixed for me with 11.3.1 on Ubuntu 23.10! 🙂

**Comment by rmcrackan** [comment]
Thank you for verifying. I know 11.3.1 is a 'pre-release' version, not a full release yet, but I'm going to close this ticket since it's solved. Thanks again :)

**Comment by fusentasticus** [comment]
The first thing a new user will stumble on is this issue - because as of today this bug is still present in the version that's distributed and it's a show stopper. But a new user won't find the issue without changing search settings in github because closed issues are by default not presented. Just a thought :)

**Comment by rmcrackan** [comment]
> The first thing a new user will stumble on is this issue - because as of today this bug is still present in the version that's distributed and it's a show stopper. But a new user won't find the issue without changing search settings in github because closed issues are by default not presented. Just a thought :)

Sorry, I'm afraid I don't follow. Are you saying this is still an issue in v11.1 for Mac?

**Comment by fusentasticus** [comment]
Thanks for quick reply! No, it's my misunderstanding. The format has slightly changed relative to the documentation I think: there's now a field called "Export". It's filled in with "Export" and that makes it appear like the first row is a predefined named account for export purposes. So I couldn't figure out how to add my own account in a new row. There isn't an "add account" button.  I finally reckoned that the name of the first account is not "Export" and that I simply had to start typing my email address in the "Audible" field. (Perhaps put "your Audible login information" in shadowed italics to direct the user's attention?) So despite your best efforts in your guide, I sheepishly admit to having missed it :)
